.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
+# flake8: noqa
+
+
+def set_prefs(prefs):
```

Comment:
would be nice to add .vscode to gitignore

Reply:
Done. Also removed the files from the repo.